### ISSUE TITLE
Viewport does not display anything

### ISSUE THREAD

**Issue by 2SIX6** [fuse-x/studio#20]
When loading test app, hikr, the viewport does not display anything, the project remains blank. Rebuilding or refreshing do nothing, changing the viewport doesnt do anything either. Not sure whats going on here?

![2021-11-10 02_48_12-hikr - fuse X](https://user-images.githubusercontent.com/66733838/141071717-9780f6c4-df1e-4283-bd9a-62ec0b2ddef9.png)
.

**Comment by mortend** [comment]
It could look like `fuse-preview.exe` fails to start here, but I am not sure why. Maybe it is blocked somehow.

* Is this on Windows 11?

* Are you able to see the app if you run it via Uno instead of fuse X?

```
uno build --run /path/to/your/project
```

**Comment by wurin7i** [comment]
Reporting the same issue.

Win10 build 19043
Uno 2.0.1 (build 936) Windows 10 x64 4f7485a

and here is the output from uno command https://pastebin.com/kSvP78DM

**Comment by slpatech** [comment]
I am having the same problem on a HP Stream - 14-cb174wm running Windows 11 Home 64-bit Version: 22000.778. I am also not able to preview on my Android.

I have not included specific information - it does not look like there has been much activity here... however, when you want to dig in, I am here to get into the trenches with you. I will walk to the end of the earth to help.

I would love to be able to use the studio. It looks better than everything I have seen. I have used Android Studio before but want something quicker to use. The app I want to code is a budget analyzer that is sophisticated enough for very complex business needs and yet simple enough for the home user . I have a coded web app since continually updated since 2008 that I am using now and want to do justice to the app, it is slick already, and offer it for free on Android and iOS.

In other words, I am a cheerleader.

Cheers!!
